runtime/containerd/containerd.go

```diff
@@ -350,6 +350,28 @@ func (c *ContainerdRuntime) StartContainer(ctx context.Context, _ string, node *
 	return nil, nil
 }
 
+func (c *ContainerdRuntime) PauseContainer(ctx context.Context, cID string) error {
+	ctask, err := c.getContainerTask(ctx, cID)
+	if err != nil {
+		log.Debugf("container %s: %v", cID, err)
+		return nil
```

Comment:
looks like a leftover from the dev time? Should return err?

Reply:
Done. Removed the log, returning error